### PR TITLE
fixed souce code line highlighting

### PR DESCRIPTION
**Pull request by vlastavesely**
Related to #884 

My goal was to imitate behaviour of latest Nette's API and one of Doctrine ORM. There was a little problem with line numbers styling for in former versions of ApiGen number and source code has been wrapped in one HTML element and in current version this is done differently. But I solve it.

Autocomplete has been removed temporarily, for it broke the highlighting.

### CODE REVIEW

**Comment by vlastavesely** [comment]
Tested in FF and Chromium.

**Comment by TomasVotruba** [comment]
@vlastavesely Thanks a lot for the PR.

Could you please add before/after screenshot? This is related to the design and hard to see from the code.

**Comment by vlastavesely** [comment]
Now it looks like this:
screenshot


And for comparison your own screenshot how it used to look:
old

File: packages/ThemeDefault/src/@layout.latte [hunk 1]
```diff
@@ -39,11 +39,11 @@
         </div>
 
         {* jquery autocomplete *}
-        <script src="https://code.jquery.com/jquery-3.2.1.min.js" defer></script>
-        <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js" defer></script>
+        <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
```
Comment (TomasVotruba) [inline packages/ThemeDefault/src/@layout.latte]: What was the motivation to remove this?

File: packages/ThemeDefault/src/@layout.latte [hunk 2]
```diff
@@ -39,11 +39,11 @@
...
 
-        <script src="/resources/main.js"></script>
-        <script src="/elementlist.js"></script>
+        <script src="elementlist.js"></script>
+        <script src="resources/main.js"></script>
```
Comment (TomasVotruba) [inline packages/ThemeDefault/src/@layout.latte]: I recall there was some issue for path without `/`. What was the motivation to remove and switch order?

Comment (vlastavesely) [inline packages/ThemeDefault/src/@layout.latte]: switching: main.js  is going to use ApiGen.elements that has to be declared before main.js is loaded.
when path was with `/`, browser failed to load the files, for they are not really in filesystem root. Do you use Linux?

File: packages/ThemeDefault/src/@layout.latte (hunk 1 above)
Comment (vlastavesely) [inline packages/ThemeDefault/src/@layout.latte]: If I understand it correctly, this parameter forces loaded JS to be executed after page is parsed. The next two JS files, however were loaded in a standard way and so executed before jQuery has been loaded. I has problems with missing `$`.

File: packages/ThemeDefault/src/@layout.latte (hunk 2 above)
Comment (TomasVotruba) [inline packages/ThemeDefault/src/@layout.latte]: 1. I see.

2. Yes I do. I tried it locally and it works.

File: packages/ThemeDefault/src/@layout.latte (hunk 1 above)
Comment (TomasVotruba) [inline packages/ThemeDefault/src/@layout.latte]: I see, ok. I recall I had similar issue. This was just experimental feature for me, I don't really use it very often nor know a lot a about it.

File: packages/ThemeDefault/src/resources/main.js [hunk 3]
```diff
@@ -146,7 +146,7 @@ $(document).ready(function() {
             }
         }
 
-        hash = hash.join(',');
+        hash = '#' + hash.join(',');
```
Comment (TomasVotruba) [inline packages/ThemeDefault/src/resources/main.js]: ### Before

before


### After

after

**Comment by TomasVotruba** [comment]
I've added a small fix, it's ready to merge otherwise.

Tested in Chrome on Linuxu (Ubuntu 17.04)

Can I merge it?

**Comment by vlastavesely** [comment]
Yes, you can.

Your screenshots are giving us an answer to the problem with `/` in path :) You use HTTP server, where its root is exact the same with root of the generated docs. I do open the files via `file://` protocol and so, their path is never the same with filesystem's root. That is the reason why it worked for you but not for me.

**Comment by TomasVotruba** [comment]
Yep, thats the case. I think subdirectory like `localhost/api/` would fail too.

Thanks for your work :bowing_man:

**Pull request merged**